fabric-transitive-access-wideners-v1/template.accesswidener

```diff
@@ -24,4 +24,104 @@ transitive-accessible field net/minecraft/loot/LootPool bonusRolls Lnet/minecraf
+transitive-accessible class net/minecraft/village/TradeOffers$EnchantBookFactory
+transitive-accessible class net/minecraft/village/TradeOffers$SellMapFactory
+transitive-accessible class net/minecraft/village/TradeOffers$SellDyedArmorFactory
+
+# Registering custom entity spawn restrictions
+transitive-accessible method net/minecraft/entity/SpawnRestriction register (Lnet/minecraft/entity/EntityType;Lnet/minecraft/entity/SpawnRestriction$Location;Lnet/minecraft/world/Heightmap$Type;Lnet/minecraft/entity/SpawnRestriction$SpawnPredicate;)V
+
+# Creating custom block materials
+transitive-accessible method net/minecraft/block/Material$Builder lightPassesThrough ()Lnet/minecraft/block/Material$Builder;
+transitive-accessible method net/minecraft/block/Material$Builder burnable ()Lnet/minecraft/block/Material$Builder;
+transitive-accessible method net/minecraft/block/Material$Builder destroyedByPiston ()Lnet/minecraft/block/Material$Builder;
+transitive-accessible method net/minecraft/block/Material$Builder blocksPistons ()Lnet/minecraft/block/Material$Builder;
+
+# Item constructors
+transitive-accessible method net/minecraft/item/AxeItem <init> (Lnet/minecraft/item/ToolMaterial;FFLnet/minecraft/item/Item$Settings;)V
+transitive-accessible method net/minecraft/item/MusicDiscItem <init> (ILnet/minecraft/sound/SoundEvent;Lnet/minecraft/item/Item$Settings;)V
+transitive-accessible method net/minecraft/item/PickaxeItem <init> (Lnet/minecraft/item/ToolMaterial;IFLnet/minecraft/item/Item$Settings;)V
+
+# Item usage context constructors
+transitive-accessible method net/minecraft/item/ItemUsageContext <init> (Lnet/minecraft/world/World;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/util/Hand;Lnet/minecraft/item/ItemStack;Lnet/minecraft/util/hit/BlockHitResult;)V
+transitive-accessible method net/minecraft/item/ItemPlacementContext <init> (Lnet/minecraft/world/World;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/util/Hand;Lnet/minecraft/item/ItemStack;Lnet/minecraft/util/hit/BlockHitResult;)V
+
+# Text filtering
+transitive-accessible method net/minecraft/server/network/ServerPlayNetworkHandler filterText (Ljava/lang/Object;Ljava/util/function/Consumer;Ljava/util/function/BiFunction;)V
+transitive-accessible method net/minecraft/server/network/ServerPlayNetworkHandler filterText (Ljava/lang/String;Ljava/util/function/Consumer;)V
+transitive-accessible method net/minecraft/server/network/ServerPlayNetworkHandler filterTexts (Ljava/util/List;Ljava/util/function/Consumer;)V
+
+# Registering custom renderers
+transitive-accessible method net/minecraft/client/render/entity/EntityRenderers register (Lnet/minecraft/entity/EntityType;Lnet/minecraft/client/render/entity/EntityRendererFactory;)V
+transitive-accessible method net/minecraft/client/render/block/entity/BlockEntityRendererFactories register (Lnet/minecraft/block/entity/BlockEntityType;Lnet/minecraft/client/render/block/entity/BlockEntityRendererFactory;)V
```

Comment:
Fabric has an API for this to accomodate for late registration (afaict?). So, not sure if they should be added.

Reply:
Yeah I couldn't tell if the FAPI version had any inherent benefits over the vanilla one, and so figured it couldn't hurt to widen the vanilla one. If it's better for modders to use the FAPI one to avoid issues or something, then I suppose I can remove this. Will need further input.